packages/make-styles/src/shorthands/borderRadius.test.ts

```diff
@@ -0,0 +1,39 @@
+import { borderRadius } from './borderRadius';
+
+describe('borderRadius', () => {
+  it('expands to an object for a single value', () => {
```

Comment:
nit: `single` `2` `3` values for test naming seem kinda confusing

Reply:
Suggestions?

Reply:
I would go with the type of the values that are passed. For the specific case of border-radius:
- `for a given length`
- `for a top-left-and-bottom-right length and a top-right-and-bottom-left percentage`
- `for a top-left, top-right-and-bottom-left and bottom-right length`
- `for a top-left, top-right, bottom-right and bottom-left length`

This suggestion could be taken for the other tests too.

Reply:
@andrefcdias applied your suggestion in cc1f0bf 👍